.ci/pipelines/cleanup-gke.Jenkinsfile

```
@@ -26,13 +28,32 @@ pipeline {
                 retry(3)
             }
             steps {
+                script {
+                    lib = load ".ci/common/tests.groovy"
+                }
                 sh '.ci/setenvconfig cleanup/gke'
                 sh 'make -C .ci TARGET=run-deployer ci'
             }
         }
     }
 
     post {
+        unsuccessful {
+            script {
+                if (params.SEND_NOTIFICATIONS) {
+                    def msg = lib.generateSlackMessage("k8s cluster cleanup failed, manual cleanup might be required!", env.BUILD_URL)
```

`generateSlackMessage` expects 3 parameters:
https://github.com/elastic/cloud-on-k8s/blob/896a7873b36815177f2fe4a70365cde4e40c0cd9/.ci/common/tests.groovy#L16

We could make the parameter optional:

```diff
diff --git a/.ci/common/tests.groovy b/.ci/common/tests.groovy
index 22170d46..79f4e195 100644
--- a/.ci/common/tests.groovy
+++ b/.ci/common/tests.groovy
@@ -13,7 +13,7 @@ def getListOfFailedTests() {
     return result
 }
 
-def generateSlackMessage(baseMsg, URL, failedTests) {
+def generateSlackMessage(baseMsg, URL, failedTests = []) {
     def sb = new StringBuilder()
     sb.append(baseMsg)
     sb.append("\r\n")
```

Or pass an empty list:

```suggestion
                    def msg = lib.generateSlackMessage("k8s cluster cleanup failed, manual cleanup might be required!", env.BUILD_URL, [])
```